db/migrate/20201019154701_delete_lingering_subscription_contents.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteLingeringSubscriptionContents < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
```suggestion
  class SubscriptionContent < ApplicationRecord; end
  
  def change
```

This will make this migration resilient to any changes to the model class.

Reply:
Good point!